features/page-orientation.yml

```
@@ -0,0 +1,8 @@
+name: page-orientation
+description: The `page-orientation` CSS property sets the rotation of a page after it has been layed out. This allows pages with the same `size` declaration to display at different orientations.
```

```suggestion
description: The `page-orientation` CSS property sets the rotation of a page after layout for print. This allows pages with the same `size` declaration to display with different orientations.
```

I wasn't sure about `layed out` vs `laid out` so I avoided the question 😆 .

I also thought mentioning 'print' was a good idea here, but I'm not sure my suggestion is the ideal way.

---

Alternate suggestion:

```suggestion
description: The `page-orientation` CSS property sets the rotation of a page after pagination. This allows pages with the same `size` declaration to display with different orientations.
```

---

This looks good to me!